src/game.cpp

```diff
@@ -7123,7 +7123,7 @@ void game::zones_manager()
                     //Draw direction + distance
                     mvwprintz( w_zones, iNum - start_index, 32, colorLine, "%*d %s",
                                5, static_cast<int>( trig_dist( player_absolute_pos, center ) ),
-                               direction_name_short( direction_from( player_absolute_pos, center ) ).c_str() );
+                               _( direction_name_short( direction_from( player_absolute_pos, center ) ).c_str() ) );
```

Comment:
Maybe return the translated string from `direction_name` and `direction_name_short`, so we don't have to change so many files?

Reply:
Yes, and there is no need to expose the untranslated string anyway.

Reply:
I did it. Declarations have been changed to handle `warning: returning reference to temporary`.